### ISSUE TITLE
Edit on GitHub: page not found

### ISSUE THREAD

**Issue by jidanni** [OSGeo/proj.4#3751]
1. Visit https://proj.org/
2. Click "Edit on GitHub"

Page not found

**Comment by jidanni** [comment]
Alas, the fix doesn't address the real problem...

**Comment by jidanni** [comment]
Here is a current bad link vs. correct links:
```
X https://github.com/OSGeo/PROJ/edit/9.2/docs/source//docs/source/operations/projections/eqearth.rst
O https://github.com/OSGeo/PROJ/edit/9.2             /docs/source/operations/projections/eqearth.rst
O https://github.com/OSGeo/PROJ/edit/master          /docs/source/operations/projections/eqearth.rst

```
Thus there is an extra "docs/source/" in the path that needs to be removed.

**Comment by kbevers** [comment]
@mwtoews do you mind taking a look at this?

**Comment by mwtoews** [comment]
Fixed with 0908dd5cac6f48ae52e94c72d7e5ae0cf0502d72

**Comment by kbevers** [comment]
Thanks!
